src/parser/column_definition.cpp

```diff
@@ -28,8 +28,31 @@ ColumnDefinition::ColumnDefinition(string name_p, LogicalType type_p, unique_ptr
 	}
 }
 
+ColumnDefinition::ColumnDefinition(string name_p, LogicalType type_p, unique_ptr<ParsedExpression> expression,
```

Comment:
This might be better implemented as an additional parameter to the existing constructor, with a default value of false
Though like I said previously, I wonder if it's better to add this SERIAL information as ExtraTypeInfo to the LogicalType